exception_reports/reporter.py

```diff
@@ -8,30 +9,39 @@
 from html import escape
 from pathlib import Path
 from pprint import pformat, saferepr
+import platform
 
 import jinja2
 
 from exception_reports.traceback import get_logger_traceback, TracebackFrameProxy
-from exception_reports.utils import force_text
+from exception_reports.utils import force_text, gen_error_filename
 
 logger = logging.getLogger(__name__)
 
-_CURRENT_DIR = Path(__file__).parent
 
-with open(_CURRENT_DIR / 'report_template.html', 'r') as f:
-    TECHNICAL_500_TEMPLATE = f.read()
-    TECHNICAL_500_TEMPLATE = re.sub(r'\s{2,}', ' ', TECHNICAL_500_TEMPLATE)
-    TECHNICAL_500_TEMPLATE = re.sub(r'\n', '', TECHNICAL_500_TEMPLATE)
-    TECHNICAL_500_TEMPLATE = re.sub(r'> <', '><', TECHNICAL_500_TEMPLATE)
+@functools.lru_cache()
```

Comment:
minor: why cache? Isn't this only called on exception and we are going to be exiting anyway?

Reply:
We only exit for exceptions in feature collection.  This library also creates reports for ERROR level logs.  

Its possible other situations can exist where exception reports are generated but then the exception is handled and processing continues.